autonomy/cli/build_images.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2022 Valory AG
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""Build images."""
+
+
+from pathlib import Path
+from typing import Optional
+
+import click
+from aea.cli.utils.click_utils import PublicIdParameter
+from aea.configurations.data_types import PublicId
+
+from autonomy.cli.utils.click_utils import image_profile_flag
+from autonomy.configurations.loader import load_service_config
+from autonomy.constants import DEFAULT_IMAGE_VERSION
+from autonomy.data import DATA_DIR
+from autonomy.deploy.constants import DOCKERFILES
+from autonomy.deploy.image import build_image
+
+
+@click.command(name="build-images")
+@click.option(
+    "--build-dir",
+    type=click.Path(dir_okay=True),
+    help="Path to build dir.",
+)
+@click.option(
+    "--agent",
+    type=PublicIdParameter(),
+    help="Public Id of the agent to use in the docker image.",
+)
```

Comment:
@8ball030 Now you can build images without fetching service components. Just pass the public ID for the agent you want to build the image for.